ratelimiter/async.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+""" Async support for 3.5+ """
+
+import time
+import asyncio
+
+from .sync import RateLimiter
+
+
+class AsyncRateLimiter(RateLimiter):
+
+    async_support = True
```

Comment:
I think this should be removed, or do you have a use case in mind? If Python supports async the correct class is already used so why would this need to be checked?